content/en/tracing/guide/distributed_tracing.md

```diff
@@ -250,31 +250,26 @@ For a more detailed explanations of sampling and priority sampling, check the [s
 
 Priority sampling is enabled by default. To disable it, configure the `priority.sampling` flag to `false` ([see how to configure the java client][1]).
 
-
-Current Priority Values (more may be added in the future):
-
-| Sampling Value | Effect                                                                                                     |
-| -------------- | :--------------------------------------------------------------------------------------------------------- |
-| `SAMPLER_DROP` | The sampler automatically decided to not keep the trace. The Agent will drop it.                           |
-| `SAMPLER_KEEP` | The sampler automatically decided to keep the trace. The Agent will keep it. Might be sampled server-side. |
-| `USER_DROP`    | The user asked to not keep the trace. The Agent will drop it.                                              |
-| `USER_KEEP`    | The user asked to keep the trace. The Agent will keep it. The server will keep it too.                     |
-
 Manually set trace priority:
 
 ```java
 import datadog.trace.api.Trace;
 import datadog.trace.api.interceptor.MutableSpan;
-import datadog.trace.common.sampling.PrioritySampling;
+import datadog.trace.api.sampling.ForcedTracing;
 import io.opentracing.util.GlobalTracer;
 
 public class MyClass {
     @Trace
     public static void myMethod() {
         // grab the active span out of the traced method
         MutableSpan ddspan = (MutableSpan) GlobalTracer.get().activeSpan();
+
         // ask the sampler to keep the current trace
-        ddspan.setSamplingPriority(PrioritySampling.USER_KEEP);
+        ddspan.setTag(ForcedTracing.manual_KEEP);
```

Comment:
`MANUAL_KEEP`/`MANUAL_DROP`?

Assuming these have to be all caps?

Reply:
Yes, @labbati can you make the change?

Reply:
A couple of other things to fix. Doing it right now thanks!